README.md

```diff
@@ -196,6 +196,8 @@ static: certificate has an existing certificate-key pair that was previously imp
 ## Testing
 Many tests require environment variables to be set. These environment variables are translated at runtime to system properties for use by the tests. These tests will be quietly skipped if the properties they require are not set.
 
+### MQTT Tests
+
```

Comment:
The env vars listed below aren't ONLY about MQTT
```suggestion
```

You could break up the list below by topic, like it is in [aws-crt-python's README](https://github.com/awslabs/aws-crt-python/blob/main/guides/dev/README.md), and include the S3 test_helper.py directions as a sub-bullet-point of the S3 topic